packages/lego-bricks/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@webkom/lego-bricks",
+  "version": "0.0.0",
+  "description": "Component library for lego and other Abakus projects",
```

Comment:
Can't we start of with 1.0.0?

Reply:
Absolutely! I will update it before merging/releasing on npm :)

Reply:
Heh, we cannot start at 1.0.0 because I accidentally released a broken version... The first version will be 1.0.1! :)